.github/workflows/publish-cloudsmith.yml

```diff
@@ -3,11 +3,11 @@ name: Publish to Cloudsmith
 on:
   push:
     tags:
-      - v*-preview-*
+      - v*-preview.*
```

Comment:
The preview tags trigger NuGet publishes too, that fail: https://github.com/Lombiq/UI-Testing-Toolbox/actions/workflows/publish-nuget.yml Same for HL: https://github.com/Lombiq/Helpful-Libraries/actions/workflows/publish-nuget.yml